services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -328,6 +328,51 @@ public void annotationsList() {
     assertEquals("congestion,distance,maxspeed,speed", routeOptions.annotations());
   }
 
+  @Test
+  public void snappingClosuresString() {
```

Comment:
could you add a test if we set **empty** string and **empty** list via `snappingClosures()`

Reply:
done